.gitignore

```diff
@@ -9,9 +9,10 @@ daos_m.conf
 config.log
 /install/
 utils/daos_build.pyc
-hwloc-1.11.5.tar.gz
-mpi4py-2.0.0.tar.gz
-protobuf-all-3.5.1.tar.gz
+hwloc-*.tar.gz
+mpi4py-*.tar.gz
+protobuf-all-*.tar.gz
+protobuf-c-*.tar.gz
```

Comment:
I was looking at making a similar change, why not just specify *.tar.gz?

Reply:
We could change it to that. I can't think of a reason why we would want to add a tgz. Should we ask others to see if they have a reason? Although even if we did want to version a zip file we could always force add it.

Reply:
I already exclude *.tar.gz in my private git excludes, and it hasn't done me any harm yet. I can't think of any reason to add one either.

Reply:
changed it to be *.tar.gz and also *.tgz

Reply:
It seems the alternative is updating the .gitignore after something changes in components.  DAOS is pretty bad at keeping this file up to date at the best of times, it should have a pylint.log entry and script entry as well.

Reply:
What do those entries look like? I'll add them to the patch.